packages/dsh-tauri-panel/src/client/service.tsx

```
@@ -0,0 +1,213 @@
+      {icon !== undefined && <span className={PANEL_CLASSES.menuItemIcon}>{icon}</span>}
+      <span className={PANEL_CLASSES.menuItemLabel}>{children}</span>
+    </button>
+  )
+}
+
+/**
+ * 切换会话区替换：未替换则打开（conversation 槽动态注册 spec.render），
+ * 已替换则关闭恢复官方会话界面。子插件在 ActionItem 的 onClick 里调用。
+ * @param spec - 内容区规格。
+ */
+export function renderPanelContent(spec: PanelContentSpec): void {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)